src/imex/import/useImportTable.utils.ts

```diff
@@ -215,8 +215,26 @@ export const parseRawData = async <D extends { id?: string | number }>(
           }
         }
 
+        // Get the previous value for validation
+        const prevValIdentifier = get(
+          importedRowValue,
+          identifierColumn.accessor as string
```

Comment:
You should use the `getPath` function in case we have a function accessor

Reply:
And maybe try to move up prevVal section that is bellow to avoid unecerray loops ?